decko/README.rdoc

```diff
@@ -123,4 +123,7 @@ If you're new to Decko, you'll find lots more info at http://wagn.org
 If you're looking to go deep into the code, start with http://www.rubydoc.info/gems/card
 
 
+==FAQ
+You may find some useful information here[link:FAQ.md] when you encountered problems. 
```

Comment:
I think we need four things in the More Resources section that align with the four groups:
A: wagn.org for general resources
B: wagn.org/installation for installation problems (need to be able to find the imagemagick stuff from there)
C. rubydoc.info/gems/card for mod developers
D. Contributing.md for core developers.